addie/advanced/isrp_gui_table_init.py

```diff
@@ -8,12 +8,30 @@
 
 class IsRepGuiTableInitialization(object):
     _script = "/SNS/users/zjn/pytest/FOD.py"
-    _cmd = "python {}".format(_script)
+    _cmd = "/usr/bin/python {}".format(_script)
```

Comment:
Why add the `/usr/bin`?
We don't want to assume where the python exec is installed.

Reply:
I suppose addie will work with python 3. Previously I was testing it under python 2 environment and it seems some angstrom symbols were not displayed properly. So I changed to python 3 environment and things are working fine. But once I bump into python 3 env, Joerg's script won't work since that one relies on python 2. I did not figure out a proper way to go so just specify manually a specific python version for it. maybe you have a better solution for this?

Reply:
So python 2 has reached end-of-life.
It is a matter of time before it is removed from analysis cluster and only python 3 will be available.
This is something that Joerg will have to start addressing in his scripts.
@peterfpeterson may have more thoughts but I don't intend on keeping python 2 support.
Needs to be communicated to Joerg.

Yet, we don't want to keep ADDIE just working with Joerg's scripts, we also have the Mantid reduction that we support.
And we don't want to just run on analysis cluster, we have had multiple Users ask for a take-home version.
We package with conda and make it available for others to use to do so but will only be portable for the Mantid reduction.

Reply:
So in an offline talk, we realize the major issue is that the Anaconda package of of ADDIE is _only_ python 3 now.
This is due to the fact that [Mantid framework anaconda package](https://anaconda.org/mantid/mantid-framework/files) and [Mantid Workbench anaconda package](https://anaconda.org/mantid/mantid-workbench/files) dropped Python 2 support 7 months ago (due to their dependencies like matplotlib, numpy, etc. dropping support as well)

We will keep this in for now to test on analysis, submit an issue to remove once we get Joerg's scripts working in Python 3, and then change in the future. 

It will stay for now.